fvm/evm/precompiles/signature_test.go

```diff
@@ -0,0 +1,20 @@
+package precompiles_test
+
+import (
+	"testing"
+
+	gethCrypto "github.com/ethereum/go-ethereum/crypto"
+	"github.com/stretchr/testify/require"
+
+	"github.com/onflow/flow-go/fvm/evm/precompiles"
+)
+
+func TestFunctionSignatures(t *testing.T) {
```

Comment:
add test for `SplitFunctionSignature`?

Reply:
added at [a6abd9b](https://github.com/onflow/flow-go/pull/5233/commits/a6abd9b9ae7efe8c96f8b3ae11a11004bfefa3c7)